### PR TITLE
Update How do I get support?

### PR DESCRIPTION
**Pull request by bwentwor**
The "How do I get Support?" section is being updated to align with the IBM Cloud Console and to provide a direct link to the Support Center. The changes are needed for the chat assistant.

### CODE REVIEW

File: billing-faq.md [hunk 1]
```diff
@@ -139,8 +140,7 @@ If the credit isn't on the Usage page, go to **Invoices** and click link with th
 ## How do I get support?
 {: #contactsupport}
 {: faq}
-
-Click **Support** in the console menu bar to access the Support Center. From there, start with leveraging the list of common FAQs. If you don't find the answers that you need, see the **Need more help?** section to contact IBM Cloud support.
+Go to the [Support Center](https://{DomainName}/unifiedsupport/supportcenter){: external}[Support](https://{DomainName}/billing/promotions) page in the console. From there, review the list of common FAQs. If you don't find the answers that you need, see the **Contact Support** section to contact IBM Cloud Support.
```
Comment (hooznora) [inline billing-faq.md:143]: Suggestion for this section: 
You can get support by going to the [Support Center page](https://{DomainName}/unifiedsupport/supportcenter){: external} directly, or you can also access the page from the console menu bar. On the **Support Center** page you can find a list of common FAQs. If you don't find the answers that you need, see the **Contact Support** section to contact IBM Cloud Support.

File: billing-faq.md [hunk 2]
```diff
@@ -72,7 +72,8 @@ You might manage your payment method on a separate billing platform. You can eas
 
 You must have an active credit card or PayPal account on file. You can replace an existing credit card with a new one. 
 
-* As of 10 December 2020, if you're using a new US-based Pay-As-You-Go account with credit card billing, you can replace your credit card by going to the [Support Center](https://{DomainName}/unifiedsupport/supportcenter){: external} and opening a support case. 
+* As of 10 December 2020, if you're using a new US-based Pay-As-You-Go account with credit card billing, you can replace your credit card by going to the [
+Center](https://{DomainName}/unifiedsupport/supportcenter){: external} and opening a support case.
```
Comment (hooznora) [inline billing-faq.md:76]: The link here seems to be missing the "Support" part:

[Support Center](https://{DomainName}/unifiedsupport/supportcenter){: external}

Comment (bwentwor) [reply billing-faq.md:76]: This information is not true. That support case would go the account/billing team and they are not authorized to replace a credit card. In fact, if a credit card # is included in a support case, it is a serious security violation. Please remove that whole section on "As of 10 December 2020..." The customer must either add it on the Payments page: (https://{DomainName}/billing/payments)
There will either be a form to fill out on that page or they will be redirected to the IBM Wallet.

File: billing-faq.md (hunk 1 above)
Comment (bwentwor) [reply billing-faq.md:143]: @hooznora I am not that familiar with the documentation code. Please make sure that it links to the Support Center. That change can be committed.

**Comment by bwentwor** [comment]
I am closing this PR and will open a new one.

**Pull request closed without merging**